code/src/components/AlbumCard.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import ArtistName from 'components/ArtistName'
+import Icons from 'components/Icons'
+
+const AlbumCard = (props) => {
+    return (           
+        <article className='album-container'>
+            <div className='wrapper-for-children'>          
+                <div className='overlay-relative'>
+                    <img className='album-cover' src={props.newReleases.images[1].url} alt="Album Cover" />
+                    <div className='overlay'>
+                        <h2><Icons /></h2>
+                    </div>    
+                </div>
+
+                <h2><a className='album-name' target="_blank" rel="noopener noreferrer" 
```

Comment:
Great add with the "noopener noreferrer", I learned this thanks to React telling me it should be there in our project - had no idea before!